### ISSUE TITLE
Errors updating channel

### ISSUE THREAD

**Issue by marconoris** [javierarce/arena-manager#5]
When I want to update a channel that I have previously downloaded, I get a series of error messages because the plugin finds previously downloaded files and cannot overwrite them. I think it is essential to implement a form of sync that allows to update the folders when the content of the channel changes.....

**Comment by javierarce** [comment]
Can you share the messages you received? And does it happen always with any channel you download twice? Also, what platform are you using to run Obsidian?

**Comment by marconoris** [comment]
It happen always: https://capture.dropbox.com/W3YfAPSViISoIJnK (in the video there are several attempts with different channels).

Obsidian version 1.7.1 (Installer 1.6.5) on MacOS

**Comment by javierarce** [comment]
This should be fixed in the latest version. Sorry for the delay in getting back to you!

**Comment by marconoris** [comment]
Tranqui!

Hace unas semanas me puse a jugar con Claude y he montado una version beta usando API 3 de Are.na... no se si y cuando acabaré con una versión pública del plugin... si te apetece te paso el código, igual te ayuda para montar la versión 3?

**Comment by javierarce** [comment]
Claro, encantado de echarle un vistazo, aunque el plugin ya usa la versión 3.0 de la API.
